### ISSUE TITLE
Google colab integration

### ISSUE THREAD

**Issue by grst** [scverse/scverse-tutorials#17]
Would be nice to have integration with google colab in addition to binder. 
To make this work we need some hack that allow the use of conda envs with colab

**Comment by adamgayoso** [comment]
I would be concerned about versioning of jax/pytorch for GPU in colab as well as the startup cost of installing and creating a conda env.

**Comment by grst** [comment]
Yeah, that's not ideal... but what would you suggest to do? Just not
support it at all?

**Comment by adamgayoso** [comment]
I think this is fine if, for example, all scvi-tools models are loaded from disk.

However, I'm assuming we will make it easy to run these locally? In that case it would be good to have guides on how to install things locally with a GPU
